### PR TITLE
Data Analytics | Initial Setup

### PR DESCRIPTION
**Pull request by jyin1003**
Added dependencies, controller, view, and routing

### CODE REVIEW

**Comment by jyin1003** [comment]
line-chart-demo

File: app/controllers/data_vis_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class DataVisController < ApplicationController
+    
+    def data_view; end
+end
```
Comment (AwolDes) [inline app/controllers/data_vis_controller.rb:4]: Usually before an `end` there is an empty line

```suggestion
    def data_view; end

end
```

File: config/routes.rb [hunk 2]
```diff
@@ -7,4 +7,8 @@
 
   # Defines the root path route ("/")
   root "hothacks#index"
+
+  resources :data_vis
```
Comment (AwolDes) [inline config/routes.rb:11]: Does this need to be included in the routes file?

**Review by AwolDes — APPROVED** [review]
Happy with this, just a few non-blocking comments. Clean & consistent code is nice, do we need a linter?

File: config/routes.rb [hunk 3]
```diff
@@ -7,4 +7,8 @@
...
+  get "analytics" => "data_vis#data_view"
```
Comment (trace-n) [inline config/routes.rb:12]: Should we follow this route format for consistency or use eg:
get "/analytics", to: "data_vis#data_view"

**Review by AwolDes — APPROVED** [review]

**Pull request merged**